tracer/test/Datadog.Trace.Security.Unit.Tests/EncoderUnitTests.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Datadog.Trace.Security.Unit.Tests;
 
+[Collection("SecuritySequentialTests")]
```

Comment:
nit: if you want to avoid magic strings you can do
```suggestion
[Collection(nameof(SecuritySequentialTests))]
```

Reply:
True